tools/build_rules/external_tools/external_tools_toolchain.bzl

```diff
@@ -0,0 +1,23 @@
+"""Build rules used to define toolchains for external tools."""
+
+def _external_tools_toolchain_impl(ctx):
+    return [
+        platform_common.ToolchainInfo(
+            asciidoc = ctx.attr.asciidoc,
+            dot = ctx.attr.dot,
+            python = ctx.attr.python,
+            cat = ctx.attr.cat,
+            path = ctx.attr.path,
+        ),
+    ]
+
+external_tools_toolchain = rule(
+    implementation = _external_tools_toolchain_impl,
+    attrs = {
+        "asciidoc": attr.string(),
+        "dot": attr.string(),
+        "python": attr.string(),
+        "cat": attr.string(),
+        "path": attr.string(),
```

Comment:
If you're providing `path` the others shouldn't be necessary, no?

Reply:
Yeah. I left asciidoc because we call it explicitly, but removed the others.